lib/perl/Genome/Qc/Tool/VerifyBamId.pm

```diff
@@ -7,6 +7,16 @@ use List::MoreUtils qw(any);
 
 class Genome::Qc::Tool::VerifyBamId {
     is => 'Genome::Qc::Tool::WithVariationListVcf',
+    has => [
+        default_genotype_vcf_file_id => {
+            is => 'Text',
+            is_optional => 1,
+        },
+        default_genotype_vcf_file => {
+            is => 'Genome::SoftwareResult::ImportedFile',
```

Comment:
I could see a case where they might want to do that. I changed it to be a `SingleFile`